DEPRECATIONS.md

```diff
@@ -0,0 +1,20 @@
+# Deprecations
+
+## v1.6.0
+
+### stew.find
+
+(insert technical rationale that I am not fully aware of)
```

Comment:
`stew.find(path)` tried to do magic to figure out what the user wanted, the result was a more confusing and brittle experience.